engine/runtime/src/test/java/org/enso/interpreter/test/ValuesGenerator.java

```diff
@@ -0,0 +1,153 @@
+package org.enso.interpreter.test;
+
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.EnumSet;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+import java.util.Set;
+import org.graalvm.polyglot.Context;
+import org.graalvm.polyglot.Value;
+import org.junit.Assert;
+import static org.junit.Assert.assertFalse;
+import static org.junit.Assert.assertTrue;
+
+/**
+ * The purpose of this class is to generate various values needed for other
+ * tests. Users of this support class can configure what kind of values they
+ * need - Enso, Java, JavaScript, etc. and then call appropriate methods to
+ * obtain such values. It's up to the tests to use these values meaningfully.
+ */
+class ValuesGenerator {
```

Comment:
@Akirathan & others - if you need to write tests that deal with various values in the Enso runtime, consider extending this generator. If you need dates, add `List<Value> dates()`, if you need more polyglot, add JavaScript, Python, etc.